releasenotes/_deconst.json

```diff
@@ -0,0 +1,8 @@
+{
+    "contentIDBase": "https://github.com/rcbops/rpc-openstack/v13”,
+    "githubUrl": "https://github.com/rackerlabs/docs-rackspace/",
```

Comment:
We can do docs-rpc, but it is a private repo so only people with access would be able to use the link. Everyone else would get a 404. That's actually why the docs-rpc docs link to docs-rackspace for their issues, as the latter repo is public.

So if we want to keep our RPC docs issues all in one place, then leaving the githubURL as docs-rackspace is the best option.